dspace-server-webapp/src/main/java/org/dspace/app/rest/OpenSearchController.java

```diff
@@ -115,9 +133,36 @@ public void search(HttpServletRequest request,
 
             // support pagination parameters
             DiscoverQuery queryArgs = new DiscoverQuery();
-            queryArgs.setQuery(query);
+            if (query == null) {
+                query = "";
+            } else {
+                queryArgs.setQuery(query);
+            }
             queryArgs.setStart(start);
             queryArgs.setMaxResults(count);
+            queryArgs.setDSpaceObjectFilter(IndexableItem.TYPE);
+            if (sort != null) {
+                //this is the default sort so we want to switch this to date accessioned
+                if (sort.equals("id")) {
+                    queryArgs.setSortField("dc.date.accessioned_dt", SORT_ORDER.desc);
```

Comment:
@ConfusionOrb221 : Silly question perhaps, but instead of this change, shouldn't we look to see if there's a way to simply change the default sort (from `id` to `dc.date.accessioned`) in the UI code?

I just realized that we do this in the `collection-page.component.ts` here, as an example: https://github.com/DSpace/dspace-angular/blob/main/src/app/collection-page/collection-page.component.ts#L76

If we change the default `SortOptions` in the UI code, we might be able to drop this `if`. 

(Sidenote: it looks like the last build failed cause of minor Checkstyle failures)

Reply:
I wasn't sure how to exactly handle this because I'm getting the sort options for rss from the pagination options and since the default pagination is id I guess thats what it defaults to? I figured it didn't really matter where we made this change but since my angular pr was already approved I made it here

Reply:
@ConfusionOrb221 : The main reason I suggested this is that this change seems specific to RSS/Atom feeds...but you've made it here more generally for *everything related to OpenSearch*.  I guess I'm not sure if institutions are using OpenSearch directly for integrations of some sort.... if so, they might be surprised if they can no longer sort by `id` at all (as it'll always get changed to `dc.date.accessioned`).

That's why I was wondering if it'd be better to move this into RSS/Atom specific code, which implies to me that'd likely mean moving it into the UI layer.  Since this is such a small UI change, I'd just retest it myself quickly & merge it if it's all working properly.

Reply:
All that said, if we see no way to make this work at the UI layer... we can create a ticket for this known issue and fix it later.  It just seems like this current approach may not be "ideal" for all institutions.

Reply:
No I can make it work pushed a change to both pr's to allow for this just now.